frontend/src/metabase/components/AppBanner/AppBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Location } from "history";
+
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import {
+  PaymentBanner,
+  shouldRenderPaymentBanner,
+} from "metabase/nav/components/PaymentBanner/PaymentBanner";
+import { DatabasePromptBanner } from "metabase/nav/components/DatabasePromptBanner";
+
+interface AppBannerProps {
+  location: Location;
+}
+
+export const AppBanner = ({ location }: AppBannerProps) => {
```

Comment:
I don't see tests for this component. Do they exist?

Reply:
They don't exist. This component used to house what's in the `PaymentBanner` which is newly introduced in this PR. The only tests for the previously `AppBanner` aka `PaymentBanner` (now) are [E2E tests](https://github.com/metabase/metabase/blob/37a24a471908b2caf64698418614a905a80e4d66/e2e/test/scenarios/admin/subscription/payment-failure.cy.spec.js#L11).

Would it make sense to add a few assertions in that file `e2e/test/scenarios/admin/subscription/payment-failure.cy.spec.js` to assert that the database prompt banner won't be displayed, because the database prompt banner will only display when the payment banner isn't showing?

Reply:
Added tests